odoo.rst

```diff
@@ -0,0 +1,189 @@
+
+* Username: ``admin``
+* Password: ``admin``
+
+After the first login, it is strongly recommended to change the Odoo
+password by following these steps:
+
+1. Click on "Administrator" in the top right corner, then on "Preferences".
+
+2. Set your own email address.
+
+3. Open the "Account Security" tab, click on "Change password", enter the
+   current password.
+
+4. Enter and confirm the new password.
+
+The session will be closed, and you will need to log in again.
+
+
+Database management
+-------------------
+
+From the login screen, it will be possible to manage databases by clicking
+on the "Database Management" link.
+
+The currently set master password is: ``adminmaster``.
+
+The first step for the application's security will be to change the master
+password by following these steps:
+
+1. Click on "Set Master Password".
+2. Write the current password and then the new password.
+3. Click on "Continue".
+
+Once the new Master Password is set, it will be possible to:
+
+1. Perform a database backup.
+2. Duplicate the database.
+3. Create a new database.
+4. Delete the database.
+
+Note: The master password will always be necessary for the activities
+listed below.
+
+Database Backup
+^^^^^^^^^^^^^^^
+
+To perform a database backup, simply follow these steps:
+
+1. Click on "Backup".
+2. Enter the Master Password.
+3. Click on Backup.
+
+The database will be downloaded locally and can be used for restoration or
+as a basis for creating a parallel production database.
+
+Duplicate the database
+^^^^^^^^^^^^^^^^^^^^^^
+
+To create a copy of the production database, simply follow these steps:
+
+1. Click on "Duplicate".
+2. Enter the Master Password.
+3. Enter the name of the new Database.
+4. Check "Neutralize," which disables cron jobs and prevents the system
+   from sending automatic notifications.
+5. Click on Continue.
+
+The new database will be displayed below the existing one.
+
+Create a new database
+^^^^^^^^^^^^^^^^^^^^^
+
+To create a new database, simply follow these steps:
+
+1. Click on "Create Database".
+2. Enter the Master Password.
+3. Enter the name of the new Database.
+4. Enter Email and Password.
+5. Select the language.
+
+Note: The newly created database will be completely empty and devoid of
+any configuration, module, or application.
+
+Delete the database
+^^^^^^^^^^^^^^^^^^^
+
+To delete a database, simply follow these steps:
+
+1. Click on "Delete".
+2. Enter the Master Password.
+3. Click on Delete.
+
+
+Installation image
+==================
+
+The installation image was created by Innovyou for the following reasons:
+
+1. The image contains Odoo core modules, OCA modules, and proprietary
+   modules.
+
+2. The official Odoo image is not versioned beyond V16; its content is
+   overwritten periodically. This would prevent us from knowing for sure
+   which code is being used in that particular installation and
+   consequently would not allow us to provide proper support or debugging.
```

Comment:
This is more for the developer manual, I'd remove it.
I do not understand the point 2.